.github/workflows/build-docker-image.yml

```diff
@@ -82,6 +79,18 @@ jobs:
           git config --global user.email "github-automation@users.noreply.github.com"
           git config --global user.name "Github Automation"
 
+      - name: Check Platform String
+        id: check
+        run: |
+          if [ ${{ inputs.multiarch }} == true ]; then
```

Comment:
you don't really need the multiarch boolean or? you could also simply check the platform == "multiarch"

Reply:
You are right, But I would prefer to avoid checking strings.